docs/rest-api.yml

```diff
@@ -200,12 +200,12 @@ paths:
           type: string
       responses:
         '204':
-          description: The server has stopped
+          description: The user's notebook server has stopped
         '202':
-          description: The server has been asked to stop, but is taking a while
+          description: The user's notebook server has not stopped, but a stop request has been submitted
```

Comment:
This has lost the information that 202 will only happen if it is taking a while to stop. Can you add that somewhere?